src/seedu/addressbook/data/person/Name.java

```diff
@@ -37,10 +38,13 @@ public static boolean isValidName(String test) {
     }
 
     /**
-     * Retrieves a listing of every word in the name, in order.
+     * Retrieves a listing of every word in the name, in order and in lowercase letters.
      */
     public List<String> getWordsInName() {
```

Comment:
Since you are changing the logic of function, maybe even the name should change?

Also it would have been better to make a new function here for `lowerCase` cause other classes which were using this functions before might have been using it for getting exact words and not lowercase once